index.bs

```diff
@@ -1219,7 +1219,7 @@ following Web IDL.
     dictionary CollectedClientData {
         required DOMString           challenge;
         required DOMString           origin;
-        required DOMString           hashAlg;
+        required DOMString           hashAlgorithm;
```

Comment:
I'm not up to date enough to understand which instance this is but please don't use multiple strings for the same thing towards the authenticator. It's all data we have to store and we win nothing. I rather use my <256k of flash storage for code and key storage than pretty member names.